### ISSUE TITLE
unable to have two multi-root projects run in neovim

### ISSUE THREAD

**Issue by SimplyKnownAsG** [eruizc-dev/jdtls-launcher#16]
I use JDTLS from neovim LSP with a multi-root configuration. I can only get JDTLS to work with 1 multi-root-project at a time. When trying to start a second multi-root-project (in another directory), I get the following error from `:LspInfo` (follow the log...).

```
ERROR: transport error 202: bind failed: Address already in use\nERROR: JDWP Transport dt_socket failed to initialize, TRANSPORT_INIT(510)\nJDWP exit error AGENT_ERROR_TRANSPORT_INIT(197): No transports initialized [src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c:734]\n
```
My configuration looks something like this:

```lua
local ws_folders_lsp = {}
local ws_folders_jdtls = {}

for line in {"./Project1", "./Project2"} do
    table.insert(ws_folders_lsp, line);
    table.insert(ws_folders_jdtls, string.format("file://%s", line))
end

local config = {
    on_attach = vim.g.on_attach,
    cmd = {'jdtls'},
    root_dir = pwd, -- effectively the current directory (I always open a directory above the git projects)
    init_options = {
        workspaceFolders = ws_folders_jdtls,
    },
}

require('jdtls').start_or_attach(config)

for _,line in ipairs(ws_folders_lsp) do
    vim.lsp.buf.add_workspace_folder(line)
end
```

I only seem to have one cache directory.

```
$ ls ~/.cache/jdtls-workspace 
jdt.ls-java-project/
```

**Comment by eruizc-dev** [comment]
It seems JDTLS uses por 1044 by default for something, I'll be taking a look at it

**Comment by eruizc-dev** [comment]
Should be fixed in v1.1.5! Remember to upgrade jdtls-launcher by running the installation again (it may ask you to delete existing installation before proceeding). Feel free to reopen this issue in case you see any problem.

For some reason I configured it so a debugger can be attached to jdtls (that's what port 1044 was for), it should be possible to spawn multiple instances now. As a side note I noticed better performance, but that might be because I have not used my personal computer in a while.

**Comment by SimplyKnownAsG** [comment]
That works great, thanks! I was uncorrectly under the assumption that `jdtls --update` would update `jdtls` itself.
